openwisp_monitoring/device/settings.py

```diff
@@ -54,3 +54,9 @@ def get_health_status_labels():
 DEVICE_RECOVERY_DETECTION = getattr(
     settings, 'OPENWISP_MONITORING_DEVICE_RECOVERY_DETECTION', True
 )
+
+DEVICE_RESOURCES_THRESHOLDS = getattr(
+    settings,
+    'OPENWISP_MONITORING_DEVICE_RESOURCES_THRESHOLDS',
+    {'memory': 0.95, 'load': 0.90, 'disk_usage': 80},
```

Comment:
If memory is 0.95 and load is 0.90, shouldn't disk_usage be 0.8?

Reply:
Very well, the format in which I was passing value to the metric was like this. I will change it, to be more consistent.